.changeset/popular-monkeys-complain.md

```diff
@@ -0,0 +1,20 @@
+---
+"@logto/experience": minor
+"@logto/schemas": minor
+"@logto/core": minor
+---
+
+add support for `login_hint` parameter in sign-in method
```

Comment:
should we pass this to the connector?

Reply:
Per offline discussion, we don't need to carry this `login_hint` to social connector, since no users have mentioned this requirement until now.